dvc/remote/http.py

```diff
@@ -43,24 +28,26 @@ def __init__(self, repo, config):
         self.path_info = self.path_cls(url) if url else None
 
     def _download(self, from_info, to_file, name=None, no_progress_bar=False):
-        callback = None
-        if not no_progress_bar:
-            total = self._content_length(from_info.url)
-            if total:
-                callback = ProgressBarCallback(name, total)
-
         request = self._request("GET", from_info.url, stream=True)
-
-        with open(to_file, "wb") as fd:
-            transferred_bytes = 0
-
-            for chunk in request.iter_content(chunk_size=self.CHUNK_SIZE):
-                fd.write(chunk)
-                fd.flush()
-                transferred_bytes += len(chunk)
-
-                if callback:
-                    callback(transferred_bytes)
+        total = self._content_length(from_info.url)
+        if name is None:
+            name = from_info.url
+
+        with Tqdm(
+            total=total,
+            leave=False,
+            bytes=True,
+            desc_truncate=name,
+            disable=no_progress_bar,
+        ) as pbar:
+            with open(to_file, "wb") as fd:
+                for chunk in request.iter_content(chunk_size=self.CHUNK_SIZE):
+                    fd.write(chunk)
+                    fd.flush()
+                    pbar.update(len(chunk))
+            # print completed progress bar for large file sizes
+            if pbar.n > LARGE_FILE_SIZE:
+                Tqdm.write(str(pbar))
```

Comment:
@efiop think this makes sense but feel free to remove these three lines if you want

Reply:
@casperdcl yeah, let's remove it, it feels kinda weird to leave progress bars only for large files. When download/upload is done - it is done, no need to leave traces.